htdocs/fourn/ajax/getSupplierPrices.php

```diff
@@ -67,6 +67,9 @@
 	$productSupplierArray = $producttmp->list_product_fournisseur_price($idprod, $sorttouse); // We list all price per supplier, and then firstly with the lower quantity. So we can choose first one with enough quantity into list.
 	if (is_array($productSupplierArray)) {
 		foreach ($productSupplierArray as $productSupplier) {
+			if ($conf->global->DISABLE_BAD_REPUTATION_PRODUCT_PRICE && $productSupplier->supplier_reputation == "DONOTORDER")
```

Comment:
Do not use syntax
$conf->global->DISABLE_BAD_REPUTATION_PRODUCT_PRICE
but instead
getDolGlobalString('DISABLE_BAD_REPUTATION_PRODUCT_PRICE')